src/components/List/Pagination.tsx

```diff
@@ -28,42 +29,38 @@ const PaginationBar: React.FC<PaginationProps> = ({
   const handlePerPageChange = (_: any, { value }: any) =>
     updateQuery({ page: calculateNewPage(page, perPage, totalCount, value), perPage: value })
 
+  const { isMobile, viewport } = useViewport()
+
   return (
-    <Grid container>
-      <Grid.Row>
-        <Grid.Column width={5} floated="right">
-          {totalCount > perPage && (
-            <Pagination
-              // totalPages={15} // Enable this to test more pages
-              totalPages={Math.ceil(totalCount / perPage)}
-              activePage={page}
-              onPageChange={handlePageChange}
-              floated="right"
-            />
-          )}
-        </Grid.Column>
-      </Grid.Row>
-      <Grid.Row>
-        <Grid.Column textAlign="right" width={6} floated="right">
-          <div style={{ display: 'flex', justifyContent: 'flex-end', alignItems: 'center' }}>
-            <span style={{ padding: 5 }}>{`${perPageText}:`}</span>
-            <Dropdown
-              selection
-              compact
-              floated="right"
-              options={paginationPresets.map((value: number) => ({
-                key: value,
-                text: String(value),
-                value,
-              }))}
-              value={perPage}
-              onChange={handlePerPageChange}
-              style={{ minWidth: 20 }}
-            />
-          </div>
-        </Grid.Column>
-      </Grid.Row>
-    </Grid>
+    <div className="flex-column" style={{ width: '100%', alignItems: 'flex-end', gap: 10 }}>
```

Comment:
Plain `div` flex boxes much easier to deal with than these stupid `Grid` components 🙄

Reply:
All good.